db/users.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const User = require('./init').User
+
+function createUser (username, password, email) {
```

Comment:
You should check to see that username, password, and email are not null or undefined